InformationScripting/scripts/sink.py

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
True, but I think as long as it isn't a real bottleneck why not just leave it. I think queries in general should only be written in C++ if there is a good reason for it. And the round trip to python should not be noticeable for a user.